simpa/core/simulation.py

```diff
@@ -54,6 +55,11 @@ def simulate(simulation_pipeline: list, settings: Settings, digital_device_twin:
         simpa_output_path = path + settings[Tags.VOLUME_NAME]
 
     settings[Tags.SIMPA_OUTPUT_PATH] = simpa_output_path + ".hdf5"
+    
+    version_path = Path(__file__).parent / "../../VERSION"
```

Comment:
Was this tested both in a local copy of SIMPA and when using SIMPA as an external library?
We used to have issues correctly resolving local paths in the latter scienario.